api/catalog/api/constants/licenses.py

```diff
@@ -0,0 +1,23 @@
+# This differs from the frontend which does not consider CC0 as a CC license
```

Comment:
Why this difference?

Reply:
I'm not sure, but this has no difference in the outcome. The frontend puts CC0 with the public domain licenses and then in the `isCc` function adds an `OR` condition to include CC0. The backend places it with CC licenses and then doesn't need the `OR` condition. Either way, the difference is historical and the outcome is the same in both cases.

Reply:
I wish there was an easy way to unify these implementations so we don't have to maintain those differences :confused:

Reply:
I agree but changing it on the backend side would be wrong as that's a versioned API and also philosophically correct (in my non-lawyer opinion) to consider CC0 a CC license.

Reply:
Well, it's just not _always_ a license, I guess that's the distinction the frontend code chose to make: https://wiki.creativecommons.org/wiki/CC0_FAQ#How_does_it_work.3F

However, I'm a little wary that the distinction matters. It's good to have "technically correct variable names" but it's also good to have easy to understand and obvious code :slightly_smiling_face: Either way, it'd be nice to unify these implementations into a _single implementation_, not just in the actual implementation details. That's more what I meant.